### ISSUE TITLE
Bug (serious): Scrolling with stick doesn't work

### ISSUE THREAD

**Issue by cheater** [phileday/whirligig-old-issues#84]
In the latest version of WG (early access 4.8), scrolling lists with the stick doesn't work any more. I am using a standard Xbox One S controller. It worked before.

**Comment by phileday** [comment]
The stick control on the gamepad for scrolling up and down was disabled in the browser as it conflicted with arrow control that the gamepad uses. It could be re-enabled if that wasn't much of a concern.

**Comment by cheater** [comment]
Yeah, it definitely has to work, there's no other good way to scroll.

**Comment by phileday** [comment]
I'll look at reenabling it and hope no one complains.

**Comment by cheater** [comment]
works in 4.81

**Comment by cheater** [comment]
Too soon. It actually stopped working after some time. It still works in the playlist, but not in the file browser.

**Comment by phileday** [comment]
When you say stopped working, as it didn't scroll up and down at all?

**Comment by phileday** [comment]
OK. I think I've found the problem. The scroll works on the option highlighted. I've changed the way this works and it seems to be working more smoothly now.

**Comment by cheater** [comment]
When I said it stopped working, it didn't scroll up and down at all. But since then I haven't been able to replicate it, so it feels like it's okay now.

**Comment by phileday** [comment]
Thanks, hopefully the update I made helped. I close the ticket

**Comment by cheater** [comment]
hold up, don't close any ticket until the reporter has checked that it's fixed. I'll close it once it's fixed. Please reopen for now.

**Comment by phileday** [comment]
Sure.

**Comment by cheater** [comment]
seems to work! Thanks

**Comment by cheater** [comment]
i tried today and it's broken again.

**Comment by phileday** [comment]
How did it break and is it replicatable?

**Comment by cheater** [comment]
It just didn't do any scrolling at all. I'm not sure if it can be replicated always, but I just tried it and it worked the first time I opened the browser, but when i closed it and opened it again, it didn't work any more.
